model/components/prometheus.yml

```diff
@@ -0,0 +1,105 @@
+---
+components:
+
+  # prometheus
+  - id: prometheus
+    name: Prometheus
+    description: Prometheus is an open-source systems monitoring and alerting toolkit.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - agent
+    capabilities:
+      - tech:
+        - java
+        - go
+        - scala
+        - python
+        - ruby
+        - bash
+        - c++
+        - common-lisp
+        - elixir
+        - erlang
+        - haskell
+        - lua
+        - .net
+        - c#
+        - nodejs
+        - php
+        - rust
+      - aspects:
+        - metrics-app
+        - metrics-process
+
+  # prometheus Server
+  - id: prometheus-server
+    name: Prometheus Server
+    description: The Jaeger collector receives traces from Jaeger agents and runs them through a processing pipeline. Currently our pipeline validates traces, indexes them, performs any transformations, and finally stores them.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - collector
+      - visualization
+      - storage
+    connections:
+      - graphite
+      - influx-db
+      - chronix
```

Comment:
can you please remove the commercial tools for here.. Also remember this 500 stars minimum we discussed?

Reply:
ohh I thought last time you said 200 stars are ok as well :D I did remove the commercial tools?? Chronix is open-source at least they say so on their website -> http://www.chronix.io/ => `
The whole Chronix Stack is open source and free to use for everyone without any restrictions`

Reply:
I was not thinking on chronix, but just check again.. I see crate-db is commercial for example, not sure for rest of the storage integrations?

Reply:
But on their website they say CrateDB is openSource -> https://crate.io/pricing/

Reply:
ok my bad :(